### PR TITLE
Fix progress for INSERT SELECT

### PR DESCRIPTION
**Pull request by azat**
### Changelog category (leave one):
- Not for changelog (changelog entry is not required)

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
Fix progress for INSERT SELECT (before the fix for TCP protocol there was `written_rows`/`written_bytes` always 0 and for HTTP protocol all intermediate progress packets was with zero `written_rows`/`written_bytes`)

### CODE REVIEW

File: tests/queries/0_stateless/02458_insert_select_progress_tcp.python [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+CLICKHOUSE_HOST = os.environ.get('CLICKHOUSE_HOST', '127.0.0.1')
+CLICKHOUSE_PORT = int(os.environ.get('CLICKHOUSE_PORT_TCP', '900000'))
+CLICKHOUSE_DATABASE = os.environ.get('CLICKHOUSE_DATABASE', 'default')
+
+def writeVarUInt(x, ba):
+    for _ in range(0, 9):
+
+        byte = x & 0x7F
+        if x > 0x7F:
+            byte |= 0x80
```
Comment (kssenii) [inline tests/queries/0_stateless/02458_insert_select_progress_tcp.python:17]: 🤔 
may be add a comment why this it is done this way?
not obvious calculations...

Comment (azat) [reply tests/queries/0_stateless/02458_insert_select_progress_tcp.python:17]: This was a copy-paste from `02010_lc_native.python`:

```
$ gg 'byte |= 0x80'
src/IO/VarInt.h:            byte |= 0x80;
src/IO/VarInt.h:            byte |= 0x80;
src/IO/VarInt.h:            byte |= 0x80;
tests/queries/0_stateless/02010_lc_native.python:            byte |= 0x80
```

Comment (azat) [reply tests/queries/0_stateless/02458_insert_select_progress_tcp.python:17]: And neither of this code has a comment for this, so I assume that it was considered as more or less generic?

Comment (kssenii) [reply tests/queries/0_stateless/02458_insert_select_progress_tcp.python:17]: Ok..

**Comment by kssenii** [comment]
@azat looks like the new test is flaky, flaky check fails

**Comment by azat** [comment]
> looks like the new test is flaky, flaky check fails

Indeed, actually I've been waiting for this. Fixed.

**Review by kssenii — APPROVED** [review]

**Pull request merged**